CONTRIBUTING.md

```diff
@@ -0,0 +1,303 @@
+     - Try to include a code sample/test that raises the error.
+
+3. Fix a bug or develop a feature from the roadmap:
+   - We will always have an open ticket showing the current roadmap.
+   - Pick an unassigned feature (or potentially propose new one) or an open bug ticket.
+   - Follow the instructions from [Developing Kornia](#developing-kornia) in order to setup your development
+     environment and start coding.
+   - Checkout our coding conventions. See more details below.
+   - Run the test framework locally and make sure all works as expected before sending a pull request.
+   - Open a Pull Request, get the green light from the CI and get your code merged.
+
+4. Donate resources to the project:
+   - In case you are an organization/institution that want to give support, sponsor or just use the project, please
+     contact us.
+     - [opencollective.com/kornia](https://opencollective.com/kornia)
+     - [github.com/sponsors/kornia](https://github.com/sponsors/kornia)
+   - We are open to start any kind of collaboration and hear feedback from you.
+   - We pretend to provide features on demand. Reach us !
+   - Currently looking for some kind of server donation in order to test *CUDA* code. (Please contact us).
+
+# Developing Kornia
+
+In order to start to develop, please follow the steps below:
+
+1. Fork the [kornia repository](https://github.com/kornia/kornia) by clicking on the
+[fork](https://github.com/kornia/kornia/fork) button on the repository page. This will create a copy of the kornia
+repository under your GitHub account.
+
+
+2. Clone your fork of Kornia, and add the kornia repository as a remote:
+    ```bash
+    $ git clone git@github.com:<your Github username>/kornia.git
+    $ cd kornia
+    $ git remote add upstream https://github.com/kornia/kornia.git
+    ```
+
+3. Create a new branch with a meaningful name reflecting your contribution. See an example:
+    ```bash
+    $ git checkout upstream/master -b feat/foo_feature
+    # or
+    $ git checkout upstream/master -b fix/bar_bug
+    ```
+    🚨 **Do not** work on the `master` branch!
+
+4. Creating development environment
+
+    **Using kornia script**
+
+    Assuming that you are on ubuntu, with nvidia-drivers installed. In bash, source the ``path.bash.inc`` script.
+    This will install a local conda environment under ``./.dev_env``, which includes pytorch and some dependencies
+    (no root required).
+
+    ```bash
+    $ source ./path.bash.inc
+    $ python setup.py develop
+    $ python -c "import kornia; print(kornia.__version__)"
+    ```
+
+    To install, or update the conda environment run ``setup_dev_env.sh``
+
+    ```bash
+    $ ./setup_dev_env.sh
+    ```
+
+
+    **Manually setup**
+
+    Otherwise, using a virtual environment of you preference. We recommend to use
+    [conda](https://docs.conda.io/en/latest/)) because it facilitates the Pytorch setup, mainly for those who have a
+    **GPU** available.
```

Comment:
```suggestion
    Otherwise, use a virtual environment of your preference. We recommend using
    [conda](https://docs.conda.io/en/latest/) because it facilitates the Pytorch setup, mainly for those who have a
    **GPU** available.
```